macros/src/main/scala/com/avast/cactus/Converter.scala

```diff
@@ -1,20 +1,31 @@
 package com.avast.cactus
 
-import com.avast.cactus.v3.V3Converters
-import org.scalactic.Good
+import com.avast.cactus.v3.{AnyValue, V3Converters}
+import com.google.protobuf.{Message, MessageLite}
 import org.scalactic.Accumulation._
+import org.scalactic.Good
 
 import scala.annotation.implicitNotFound
 import scala.collection.JavaConverters._
+import scala.language.experimental.macros
 import scala.reflect.ClassTag
 
 @implicitNotFound("Could not find an instance of Converter from ${A} to ${B}, try to import or define one")
 trait Converter[A, B] {
   def apply(fieldPath: String)(a: A): ResultOrError[B]
 }
 
+@implicitNotFound("Could not find an instance of AnyValueConverter from ${Gpb}, try to import or define one")
+trait AnyValueConverter[Gpb] {
```

Comment:
What is `AnyValue`? I do not see in defined anywhere. Why do we need a special converter for it?

Reply:
`AnyValue` is holder for GPBv3 ANY feature. I was not able to get it running without own converter, because from type-system POV it's just a case class.